src/Components/Aspire.Azure.Search.Documents/Aspire.Azure.Search.Documents.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>$(ComponentAzurePackageTags) azure search</PackageTags>
+    <Description>A client for Azure Search that integrates with Aspire, including logging and telemetry.</Description>
```

Comment:
```suggestion
    <Description>A client for Azure AI Search that integrates with Aspire, including logging and telemetry.</Description>
```

?

Reply:
Azure Search was just more for consistency with the extension methods, i.e. `AddAzureSearch` as per [suggestion](https://github.com/dotnet/aspire/pull/2070#discussion_r1488536192) by @tg-msft.

Reply:
API names should aim for longevity.  READMEs and docs should include product branding so it's clear you're definitely using the right library for a specific service.  Azure has [a lot of similar sounding services.](https://azure.microsoft.com/en-us/products)  When in doubt about conflicting branding between the marketing page/REST docs/whatever else, always prefer the name currently used in the Portal.

@eerhardt - do you think it'd be worth adding package tags for Azure Search, Azure AI Search, Azure Cognitive Search, Azure Search Documents, etc., so it's easier to find on nuget?  Or do you think that'll be unnecessary with the curated "Add Component" experience in VS?

Reply:
> do you think it'd be worth adding package tags for Azure Search, Azure AI Search, Azure Cognitive Search, Azure Search Documents, etc., so it's easier to find on nuget?

Yes I think it is valuable.